ModPatches/Vanilla Factions Expanded - Pirates/Defs/Vanilla Factions Expanded - Pirates/Ammo/Ammo_Mods.xml

```diff
@@ -85,4 +94,62 @@
 		</comps>
 	</ThingDef>
 
+	<!-- Bioferrites -->
+	<ThingDef ParentName="Base50mmRocket">
+		<defName>Bullet_50mmRocket_HEDF</defName>
+		<thingClass>CombatExtended.ProjectileCE_Explosive</thingClass>
+		<label>50mm direct fire rocket (HE)</label>
+		<graphicData>
+			<texPath>Things/Projectile/Rocket/70mmAPKWS</texPath>
+			<graphicClass>Graphic_Single</graphicClass>
+			<drawSize>0.70</drawSize>
+		</graphicData>
+		<projectile Class="CombatExtended.ProjectilePropertiesCE">
+			<damageDef>Bomb</damageDef>
+			<damageAmountBase>147</damageAmountBase>
+			<explosionRadius>2.5</explosionRadius>
+			<speed>80</speed>
+			<soundAmbient>RocketPropelledLoop_Small</soundAmbient>
+			<dropsCasings>false</dropsCasings>
+			<flyOverhead>false</flyOverhead>
+			<shellingProps>
```

Comment:
Since it's a direct-fire projectile, this field and its subnodes can go away. They're only for firing projectiles between maps.

Reply:
switched to a copy of heavy bolter ammo, also uploaded texture for them.